tests/console/yast2_dns_server_service_abort_keep.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
```

Comment:
might I suggest something less cryptic and in line with the rest of the tests?
`yast2_dns_server_service keep_inactive_disabled`
I believe the abort is the same like if we close the windows, so we might omit the method we abort, wdyt?

Reply:
updated.